form/unit_testing.rst

```diff
@@ -173,29 +192,22 @@ allows you to return a list of extensions to register::
 
     // ...
     use AppBundle\Form\Type\TestedType;
-    use Symfony\Component\Form\Extension\Validator\ValidatorExtension;
-    use Symfony\Component\Validator\ConstraintViolationList;
-    use Symfony\Component\Validator\Validator\ValidatorInterface;
 
     class TestedTypeTest extends TypeTestCase
     {
-        private $validator;
-
         protected function getExtensions()
         {
-            $this->validator = $this->createMock(ValidatorInterface::class);
-            $this->validator
-                ->method('validate')
-                ->will($this->returnValue(new ConstraintViolationList()));
-
             return array(
-                new ValidatorExtension($this->validator),
+                new MyFormExtension(),
             );
         }
 
         // ... your tests
     }
 
+It is also possible to load custom form types, form type extensions or type guessers using the
+``getTypedExtensions``, ``getTypes`` and ``getTypeGuessers`` methods.
```

Comment:
This should be part of a separate pull request as we will have to merge it into the `master` branch (the rest can be merged into the docs for Symfony 2.7).